app/src/components/common/PasswordValidationSchema.tsx

```diff
@@ -0,0 +1,40 @@
+import {object, string} from 'yup';
+
+export const validationSchema = object().shape({
+  email: string().email('Invalid email').required('email is required'),
+  password: string()
+    .min(8, {length: 'password must be at least 8 characters'})
+    .matches(/^(?=.*[0-9])/, {
+      message: {number: 'password must contain at least one number'},
+    })
+    .matches(/^(?=.*[a-z])/, {
+      message: {lowercase: 'Must contain at least one lowercase character'},
+    })
+    .matches(/^(?=.*[A-Z])/, {
+      message: {uppercase: 'Must contain at least one uppercase character'},
+    })
+    .matches(/^(?=.*[!@#%&])/, {
+      message: {
+        special: 'password must contain at least one special character',
```

Comment:
Can we change line 18 to "password must contain these special characters (!@#%&)"